server/controllers/applications.coffee

```diff
@@ -438,14 +438,14 @@ module.exports =
             req.application.updateAccess data, (err) ->
                 # Start application
                 manager.start req.application, (err, result) ->
-                    if err and err isnt "Not enough Memory"
+                    if err and err isnt localizationManager.t "not enough memory"
                         delete startedApplications[req.application.id]
                         return markBroken res, req.application, err
                     else if err
                         delete startedApplications[req.application.id]
                         data =
                             errormsg: err
-                            state: 'stopped'
+                            state: localizationManager.t 'stopped'
```

Comment:
Have you verified this state is a string displayed to the user and not an internal key ? Not sure